src/libsystemd-network/dhcp-network.c

```diff
@@ -19,9 +19,9 @@
 #include "unaligned.h"
```

Comment:
can you extend the commit msg here? i.e. move the explanatory stuff from the PR description into the commit descriptions please? (applies here and to the other commits a bit, too)